csrankings-v.csv

```diff
@@ -272,7 +272,6 @@ Vincent François-Lavet,VU Amsterdam,http://vincent.francois-l.be,n12uNYcAAAAJ
 Vincent Gramoli,University of Sydney,http://rp-www.cs.usyd.edu.au/~gramoli/php,Fv7ZfN8AAAAJ
 Vincent Hellendoorn,Carnegie Mellon University,https://vhellendoorn.github.io,PfYrc5kAAAAJ
 Vincent Heuveline,Heidelberg University,https://emcl.iwr.uni-heidelberg.de/people/heuveline-vincent,xsAahYgAAAAJ
-Vincent Immler,Oregon State University,https://eecs.oregonstate.edu/people/immler-vincent,YMWQhF0AAAAJ

```

Comment:
Are you certain this person does not meet the eligibility criteria (can solely advise a PhD student for a PhD in Computer Science)?

Reply:
Yes, per our school's rules, ECE faculty can solely advise ECE PhD students, and CS faculty can solely advise CS PhD students (of course, ECE faculty can jointly advise CS PhD students with CS faculty, and vice versa); Vincent is ECE: https://eecs.oregonstate.edu/people/immler-vincent